clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -318,7 +318,8 @@ Changes in existing checks
 
 - Improved :doc:`modernize-use-using
   <clang-tidy/checks/modernize/use-using>` check to fix function pointer and
-  forward declared ``typedef`` correctly.
+  forward declared ``typedef`` correctly. Ignore ``typedef`` declaration in
```

Comment:
Actually release notes should say that new option IgnoreExternC were added to control whatever typedefs in extern C should be ignored.